### ISSUE TITLE
[Release] Symja 2.0.0

### ISSUE THREAD

**Issue by HannesWell** [axkr/symja_android_library#478]
We are planning to release Symja 2.0.0 on the 12/13th March 2022.

## Release Preparation
- [x] Resolve and close all pending issues intended for this release
  - [x] Ensure all issues resolved in this release have the corresponding milestone assigned
- [x] Add missing entries in the CHANGELOG.md for this release under the `[Unreleased]` headline (ideally this was already done when the corresponding change was made).
- [x] Run the code-generator (like in 575ce353ecc6d1fe96c6d1ef32a107a98a0c77fc)

### Stabilization week
- [x] Announce the intent to release as loud as possible approximatly one week before the planned release date and ask users to test the current release candidate snapshot:
```
We are planning to release Symja 2.0.0 in one week on the 12/13th March 2022.
Please test the current SNAPSHOT and report any new bugs and regressions you found.
The release process is tracked in Symja's GitHub repository: https://github.com/axkr/symja_android_library/issues/478
```
During the stabilization week the main code as well as the build procedure (everything that affects the published artifacts) should not be touched to not accidentally introduce new bugs. However enhancements of documentation or tests may be submitted.
If major bugs/regressions are found that have to be fixed, the stabilization week should be restarted to ensure no new bugs have be introduced (maybe with shorter time-frame, but with another announcements to ask users to test the new RC again).

### Release publication
- [x] Run the `Release to Maven-Central` workflow
  - [x] Release the staged artifacts from the OSSRH staging repository into the Central Maven Repository
  https://central.sonatype.org/publish/release/#locate-and-examine-your-staging-repository

**Comment by HannesWell** [comment]
Some of the tasks are not performed for this "initial" release to Maven-Central but this is used as a template for later releases.

**Comment by HannesWell** [comment]
Symja 2.0.0 was released on 12.03.2022.
